src/mqueue.rs

```diff
@@ -100,7 +100,9 @@ impl MqAttr {
     ///
     /// - `mq_flags`:   Either `0` or `O_NONBLOCK`.
     /// - `mq_maxmsg`:  Maximum number of messages on the queue.
+    ///   The maximum value allowed by the system can be obtained from `/proc/sys/fs/mqueue/msg_max`.
     /// - `mq_msgsize`: Maximum message size in bytes.
+    ///   The maximum value allowed by the system can be obtained from `/proc/sys/fs/mqueue/msgsize_max`.
```

Comment:
These two comments are Linux-specific.  But the API is cross-platform.  So it's not really appropriate to put these here.